tests/location_tests.cpp

```diff
@@ -0,0 +1,29 @@
+/// @copyright See `LICENSE` in the root directory of this project.
+
+#include <antler/project/location.hpp>
+
+#include <catch2/catch.hpp>
+
+TEST_CASE("Testing location clone") {
+   using namespace antler::project;
+
+   antler::system::fs::remove_all("./foo");
+
+   location::clone_github_repo("antelopeio", "antler-proj", "main", 10, "./foo/foo2");
+   CHECK(antler::system::fs::exists("./foo/foo2/.git"));
+
+   //location::clone_git_repo("https://github.com/larryk85/cturtle", "main", 10, "./foo/foo3");
+   //CHECK(antler::system::fs::exists("./foo/foo3/.git"));
```

Comment:
please remove if commented out code is unnecessary

Reply:
Re-enabled in 22e0e0f